cmd/template/nodepool/provider/aws.go

```diff
@@ -33,6 +34,10 @@ func WriteAWSTemplate(out io.Writer, config NodePoolCRsConfig) error {
 		return microerror.Mask(err)
 	}
 
+	if config.MachineDeploymentSubnet != "" {
+		crs.AWSMachineDeployment.Annotations[annotation.AWSSubnetSize] = config.MachineDeploymentSubnet
+	}
```

Comment:
I was not sure where to add this logic but I did not find in apiextensions the way to configure it